test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2752,6 +2876,17 @@ static int crypto_init(odp_instance_t *inst)
 		return -1;
 	}
 
+	if (odp_schedule_capability(&sched_capa)) {
```

Comment:
Sched_capa call is not needed. Capa is set and not used.

It seems that odp_crypto_capability() call under could be removed also. Algorithm tests use is anyway.

Reply:
Agreed. Removed capability calls for sched and crypto from crypto_init() in v3.